### PR TITLE
Support Open Graph

### PR DESCRIPTION
**Pull request by birkholz**
Add Open Graph meta tags for rich links in social apps. This is just a first iteration.

There are several things I'd like to do further:
* Add a proper icon for Linty itself (awaiting design)
* Add better descriptions to builds, perhaps containing a summary of results.

Example:
screen shot 2016-12-08 at 6 02 28 pm

### CODE REVIEW

File: interface/templates/base.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
 <head>
     <title>{% if DEBUG %}[DEBUG] {% endif %}{% block title %}{% endblock %}</title>
     <meta charset="UTF-8">
-    <meta name="description" content="Free automated code linting">
     <meta name="keywords" content="code,lint,linty,pep8,pycodestyle,eslint,ci">
-    <meta name="author" content="https://github.com/birkholz">
```
Comment (jadengore) [inline interface/templates/base.html]: @birkholz Would it be helpful to link author to ZC?

File: interface/templates/interface/build_detail.html [hunk 2]
```diff
@@ -5,9 +5,14 @@
...
 {% block extra_head %}
-{% if object.status == 'pending' %}
-<meta http-equiv="refresh" content="10" >
-{% endif %}
+    {% if object.status == 'pending' %}
+    <meta http-equiv="refresh" content="10" >
+    {% endif %}
+    <meta property="og:title" content="Linty - {{ repo.full_name }}:{{ object.short_sha }}">
+    <meta property="og:description" content="{{ repo.full_name }}:{{ object.short_sha }} {% if object.status == 'success' %}passed linting!{% elif object.status == 'pending' %}is still being linted.{% else %}failed linting.{% endif %}">
+    <meta property="og:image" content="{% if object.status == 'success' %}{% static 'img/pass.png' %}{% elif object.status == 'pending' %}{% static 'img/pending.png' %}{% else %}{% static 'img/fail.png' %}{% endif %}">
```
Comment (jadengore) [inline interface/templates/interface/build_detail.html]: @birkholz Would it be better to build this content string in Python, or is this fine for template? All are pretty long

File: interface/templates/base.html (hunk 1 above)
Comment (birkholz) [inline interface/templates/base.html]: The author tag is usually intended for articles. It didn't make sense to include it here.

**Comment by birkholz** [comment]
[Add HTML meta tags for rich links](https://trello.com/c/G1hpBwRS/49-add-html-meta-tags-for-rich-links)

**Pull request merged**